src/v2/view-builder/views/password/PasswordPolicyUtil.js

```diff
@@ -0,0 +1,29 @@
+import { View } from 'okta';
+import hbs from 'handlebars-inline-precompile';
+
+const generatePasswordPolicyHtml = function(form, rulesList, prepend) {
```

Comment:
Since we are doing this and it's new to v2.. can we enhance this pretty old style of showing password requirements.. 

As use types in we should update the requirements are being met or not.. by update color of the strings to red or green.

Tech debt can be solved in a new ticket..

Reply:
Ticket created: https://oktainc.atlassian.net/browse/OKTA-484798